assets/src/settings-page/index.js

```diff
@@ -132,13 +191,48 @@ function Root() {
 				saveOptions();
 			} }>
 				<TemplateModes />
-				<h2>
+				<h2 id="advanced-settings">
 					{ __( 'Advanced Settings', 'amp' ) }
 				</h2>
 				<MobileRedirection />
```

Comment:
Why not put the ID for the section here instead of inside of the component?

Reply:
Yeah, I like having all the IDs at this level. Updated in 5e7b5de0734c92a5ab0f163656ee6abf21af96e6